filetype/types/archive.py

```diff
@@ -644,16 +644,41 @@ class Zstd(Type):
     """
     MIME = 'application/zstd'
     EXTENSION = 'zst'
+    MAGIC_SKIPPABLE_START = 0x184D2A50
+    MAGIC_SKIPPABLE_MASK = 0xFFFFFFF0
 
     def __init__(self):
         super(Zstd, self).__init__(
             mime=Zstd.MIME,
             extension=Zstd.EXTENSION
         )
 
+    @staticmethod
+    def _to_little_endian_int(buf):
+        return int(bytearray(reversed(buf)).hex(), 16)
```

Comment:
I don't know the coding standards for filetype but wouldn't it be easier to use `struct.unpack()` here?